src/python/grpcio_channelz/grpc_channelz/v1/_servicer.py

```diff
@@ -24,7 +24,10 @@ class ChannelzServicer(_channelz_pb2_grpc.ChannelzServicer):
     """Servicer handling RPCs for service statuses."""
 
     @staticmethod
-    def GetTopChannels(request, context):
+    def GetTopChannels(
```

Comment:
This is a meta note for all ancillary packages. Please manually test to make sure that introducing a typing error causes a CI failure.